lib/fog/dns/openstack.rb

```diff
@@ -1,6 +1,9 @@
 module Fog
   module DNS
     class OpenStack < Fog::Service
+      autoload :V1, File.expand_path('../openstack/v1', __FILE__)
```

Comment:
The `__FILE__` is not needed (seems an old practice) and could be replaced with:
```ruby
autoload :V1, 'lib/fog/dns/openstack/v1'
```

And same for all following `autoload`

Reply:
Did not know about this, will check it out. =)